torch/_subclasses/fake_tensor.py

```diff
@@ -305,6 +308,12 @@ def from_tensor(t, fake_mode):
     def __repr__(self):
         return f"FakeTensor({self.fake_device}, {self.size()}, {self.dtype})"
 
+    def new_empty(self, shape, dtype=None):
+        return torch.empty(shape, dtype=dtype)
+
+    def stride(self):
+        return symbolic_shapes.create_contiguous(self.shape)
```

Comment:
TODO: make stride directly support symbolic

Reply:
I'll add a comment here, but the specific reason this needs to be a method overload is that returning something from `aten.stride` in `__torch_dispatch__` requires it to be a list of ints, and so it ends up specializing all of the returned values.

Reply:
so we need aten.sym_stride

Reply:
yeah, I think it's in one of Brian's PRs